chat-commands.js

```diff
@@ -876,7 +876,7 @@ exports.commands = {
 			this.sendReplyBox(Chat.html`The room description is: ${room.desc}`);
 			return;
 		}
-		if (!this.can('declare')) return false;
+		if (!this.can('roommod')) return false;
```

Comment:
Side-note: this would be a broken permission

ROs cannot ``roommod`` as a global permission.  This change should have been ``if (!this.can('roommod', null, room)) return;`` - but as already said, this is an unwanted change.